fault/system_verilog_target.py

```diff
@@ -413,6 +413,19 @@ def make_loop(self, i, action):
         self.add_decl('integer', action.loop_var, exist_ok=True)
         return super().make_loop(i, action)
 
+    def make_join(self, i, action):
+        code = ["fork"]
+        for p in action.processes:
+            code += self.make_block(i, None, None, p.actions, p.name)
+        # join
+        if action.join_type == actions.JoinType.Default:
+            code += ["join"]
+        elif action.join_type == actions.JoinType.None_:
+            code += ["join_none"]
+        elif action.join_type == actions.JoinType.Any:
+            code += ["join_any"]
```

Comment:
We can add an internal assertion here just in case someone adds a new join_type and forgets to add support here in the target.
```suggestion
            code += ["join_any"]
 else:
     assert False, f"Unexpected joint_type: {action.join_type}"
```